internal/memory/backend.go

```diff
@@ -79,22 +79,22 @@ type backendStore struct {
 	distClient         backendv1beta1connect.DistributionServiceClient
 	store              *store
 	repoKey            *backendv1beta1.RepositoryKey
-	apikey, sessionKey string
+	apikey, sessionkey string
```

Comment:
What was the reason to rename it? To make consistent with `apikey`? But there is also `repoKey` above :)

Reply:
updated to camel case to be consistent